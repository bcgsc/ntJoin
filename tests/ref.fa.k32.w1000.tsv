test	12581791121062922:800 4831469853080092:1575 35894821415324:1630 5498242532265744:1994 19266591638982732:2656 1623192168577435:3156 3349786139723574:3439
