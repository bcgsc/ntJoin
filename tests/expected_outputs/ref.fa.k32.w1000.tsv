test	3714041376220621505:800 8162189927378643732:1575 10820188111283998344:1630 1177713728801312737:1994 17184023496183651984:2656 4671501941577321508:3156 16743415676028282381:3439
